src/Farmer/Builders/Builders.Storage.fs

```diff
@@ -106,6 +110,9 @@ type StorageAccountBuilder() =
     [<CustomOperation "static_website_error_page">]
     member _.StaticWebsiteErrorPage(state:StorageAccountConfig, errorPage) =
         { state with StaticWebsite = state.StaticWebsite |> Option.map(fun staticWebsite -> {| staticWebsite with ErrorPage = Some errorPage |}) }
+    /// Enables support for hierarchical namespace, also known as Data Lake Storage Gen2.
+    [<CustomOperation "use_hns">]
```

Comment:
I would suggest renaming this from `use_hns` to e.g. `enable_data_lake_support` or something similar.